packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -8,6 +8,21 @@ metadata:
 {{ include "logging-operator.labels" . | indent 4 }}
 spec:
   controlNamespace: {{ .Release.Namespace }}
```

Comment:
Do we need to specify a linux agent as well>

Reply:
We do not. @ahma might be able to clarify, but as far as my testing has gone, `nodeAgents` and `fluentbit` are reconciled independently from one another.

Source: https://github.com/banzaicloud/logging-operator/blob/68ff31aa0aafa2e93fc273d3304aeec4a7068f67/controllers/logging_controller.go#L230-L232
Full PR: https://github.com/banzaicloud/logging-operator/pull/665/files